notebooks/00_HubQuickStart.ipynb

```diff
@@ -1,10 +1,67 @@
 {
```

Comment:
<p>I suppose the first sentence is redundant with my comments about the first cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/spyglass-demo/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>